{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/settings.py

```diff
@@ -129,3 +132,31 @@
         'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly'
     ]
 }
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': True,
+    'formatters': {
+        'json': {
+            '()': 'pythonjsonlogger.jsonlogger.JsonFormatter',
+            'format': '{levelname} {asctime} {message}',    
+            'style': '{',
+        },
+    },
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+            'formatter': 'json',
+        },
+    },
+    'loggers': {
+        'django': {
+            'handlers': ['console'],
+            'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
+        },
+        '': {
+            'handlers': ['console'],
+            'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
+        }
```

Comment:
Looking at the default loggers defined by Django, I'd suggest to simply overwrite the [existing loggers](https://github.com/django/django/blob/22b0b73c7732ba67db4e69fd9fa75aad84c8e5c4/django/utils/log.py#L54-L62) (`django`, `django.server`) to use the updated console handler as you have defined it. 

What does the empty logger config does ?

Reply:
The empty, from what i read, should cover any undefined loggers created. So if the user makes a new logger instance with a different name it will get picked up by this.

Reply:
I added the django.server so that will get formatted too.